consumer.go

```diff
@@ -91,10 +105,23 @@ func StartConsumer(config ConsumerConfig) (c *Consumer, err error) {
 		conns: make(map[string](chan<- Command)),
 	}
 
-	go c.run()
 	return
 }
 
+func StartConsumer(config ConsumerConfig) (c *Consumer, err error) {
+	c, err = NewConsumer(config)
+	if err != nil {
+		return
+	}
+
+	c.Start()
+	return
+}
+
+func (c *Consumer) Start() {
```

Comment:
Ah, good thinking. I think I would just panic like this library does if you call `Finish()` or `Requeue()` multiple times. I'll add that now!